commit 9be341edebf5880a53ff8b27d05c7d229797c4be
Author: John Crispin <john@phrozen.org>
Date:   Mon Feb 20 10:29:43 2017 +0100

    MIPS: pci: Remove KERN_WARN instance inside the mt7620 driver
    
    commit 8593b18ad348733b5d5ddfa0c79dcabf51dff308 upstream.
    
    Switch the printk() call to the prefered pr_warn() api.
    
    Fixes: 7e5873d3755c ("MIPS: pci: Add MT7620a PCIE driver")
    Signed-off-by: John Crispin <john@phrozen.org>
    Cc: Ralf Baechle <ralf@linux-mips.org>
    Cc: linux-mips@linux-mips.org
    Patchwork: https://patchwork.linux-mips.org/patch/15321/
    Signed-off-by: James Hogan <jhogan@kernel.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/arch/mips/pci/pci-mt7620.c b/arch/mips/pci/pci-mt7620.c
index 90fba9bf98da..27ac00c36bc0 100644
--- a/arch/mips/pci/pci-mt7620.c
+++ b/arch/mips/pci/pci-mt7620.c
@@ -121,7 +121,7 @@ static int wait_pciephy_busy(void)
 		else
 			break;
 		if (retry++ > WAITRETRY_MAX) {
-			printk(KERN_WARN "PCIE-PHY retry failed.\n");
+			pr_warn("PCIE-PHY retry failed.\n");
 			return -1;
 		}
 	}
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1044_9be341edebf5880a53ff8b27d05c7d229797c4be.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1044_9be341edebf5880a53ff8b27d05c7d229797c4be.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
